.github/workflows/github-page-publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Github Page Publish
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - name: Install and Build 🔧 
+        run: |
+          npm install
+          npm run build:docs
+
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@4.1.0
+        with:
+          branch: gh-pages
+          folder: docs
```

Comment:
It seems to be simpler than the deployment script using [gh-pages](https://github.com/tschaub/gh-pages) in CodePair. 👍 
https://github.com/yorkie-team/yorkie-codepair/blob/main/.github/workflows/ghpages-publish.yml#L17-L21

Reply:
I'm curious if it is okay to use this github-pages-deploy-action rather than using this script(https://github.com/yorkie-team/yorkie-codepair/blob/main/.github/workflows/ghpages-publish.yml#L17-L21) 🤔 
And I'm also curious about which domain we are going to host this page 🤔

Reply:
Currently, we use Nginx's Reverse Proxy(EC2) to serve static pages uploaded to `gh-pages` in one domain(`yorkie.dev`).

https://github.com/yorkie-team/devops/blob/main/nginx.conf#L12-L33

Before we work on improving the static page distribution, it would be good to first distribute it in the same way.

```
Docs: https://yorkie.dev
Agent API: https://yorkie.dev/api
CodePair: https://yorkie.dev/codepair
JS SDK Reference: https://yorkie.dev/docs/main/js-sdk/reference
```